Test.Integration.NetFx48/App.config

```diff
@@ -0,0 +1,28 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<configuration>
+	<appSettings>
+		<add key="ApiKey" value="ccd68efbe25809bc" />
+		<add key="AccountId" value="5f053f004025ca08a187fad3" />
+		<add key="MerchantId" value="cf51f0ec-6078-46e9-a796-700af25e668c" />
+		<add key="Id" value="cf51f0ec-6078-46e9-a796-700af25e668c" />
+		<add key="UserId" value="haneeshv@exalture.com" />
+		<add key="SendTo" value="binishb@exalture.com" />
```

Comment:
please, do not use real email, feel free to come up with any random email that passes validation

Reply:
@binish-exalture pls, take a look